test/unit/controllers/Irc_EntryController.test.js

```diff
@@ -97,34 +75,79 @@ describe('INTEGRATION Irc_EntryController', () => {
             .expect(400)
       );
     });
+    describe('options', () => {
+      it('should return the event schema for an options request', () =>
+        request(sails.hooks.http.app)
+          .options('/irc_entry/event')
+          .expect(200)
+          .expect((res) => expect(res.body.data).to.eql(IrcEntryEventValidatorService.schema))
+      );
+    })
+    describe('post', () => {
+      var fake_request_body;
+      var detainee;
+      before(() => {
+        fake_request_body = {
+          timestamp: new Date(),
```

Comment:
you'd want to have `new Date().toString`  or something similar so that the payload is similar to what will happen in reality